imap_processing/glows/l0/decom_glows.py

```diff
@@ -49,32 +49,40 @@ def decom_packets(
         f"{imap_module_directory}/glows/packet_definitions/GLX_COMBINED.xml"
     )
 
-    packet_definition = definitions.XtcePacketDefinition(xtce_document)
+    packet_definition = spp.load_xtce(xtce_document)
 
     histdata = []
     dedata = []
 
     filename = packet_file_path.name
 
     with open(packet_file_path, "rb") as binary_data:
-        glows_packets = packet_definition.packet_generator(binary_data)
-
-        for packet in glows_packets:
+        for binary_packet in spp.ccsds_generator(binary_data):
+            packet = packet_definition.parse_bytes(binary_packet)
             apid = packet["PKT_APID"]
             # Do something with the packet data
             if apid == GlowsParams.HIST_APID.value:
-                values = [item.raw_value for item in packet.user_data.values()]
-                hist_l0 = HistogramL0(
-                    __version__, filename, CcsdsData(packet.header), *values
-                )
+                values = [
+                    item.raw_value for i, item in enumerate(packet.values()) if i > 6
+                ]
+                header = {
+                    key: value
+                    for i, (key, value) in enumerate(packet.items())
+                    if i <= 6
+                }
```

Comment:
In XTCE parsing there is no notion of header items, everything is just an item. So we have to know implicitly that we are parsing a CCSDS packet here and get the first 7 items if that is how we want to separate it out. i.e., the `.header` and `.user_data` attributes are now deprecated.